Added required caps.

g2pmas opened the pull request:
Hi dear.
I've added some bypass capacitor as suggested on ws2812b DataSheet.

G-Pereira commented:
Hi @g2pmas thank you for your contribution! It seems your fork is _"13 commits behind sphawes:mobo-rev03"_ you need to update your fork first before working on a new feature or fix so it won't require merging KiCAD files which is too complex to handle.
Since this fix can be done quickly I suggest that you update your fork by adding this repository as a new git remote which you can pull the latest commits from.

G-Pereira reviewed (changes requested):
Hi @g2pmas thank you for your contribution! It seems your fork is _"13 commits behind sphawes:mobo-rev03"_ you need to update your fork first before working on a new feature or fix so it won't require merging KiCAD files which is too complex to handle.
Since this fix can be done quickly I suggest that you update your fork by adding this repository as a new git remote which you can pull the latest commits from.

g2pmas commented:
Ok. I redo all the stuff.
Added required capacitors.
According to issue #87 i added a JST vertical connector on bottom layer and 3d model of this last.
Ready to a PR from my fork.
